common/util.go

```diff
@@ -912,71 +912,21 @@ func DurationToDays(d time.Duration) int32 {
 	return int32(d / (24 * time.Hour))
 }
 
-// DurationToHours converts time.Duration to number of hours
-func DurationToHours(d time.Duration) int64 {
-	return int64(d / time.Hour)
-}
-
-// DurationToMinutes converts time.Duration to number of minutes
-func DurationToMinutes(d time.Duration) int64 {
-	return int64(d / time.Minute)
-}
-
 // DurationToSeconds converts time.Duration to number of seconds
 func DurationToSeconds(d time.Duration) int64 {
 	return int64(d / time.Second)
 }
 
-// DurationToMilliseconds converts time.Duration to number of milliseconds
-func DurationToMilliseconds(d time.Duration) int64 {
```

Comment:
Hmmm. I we can create a generic function:
```
func DurationToUnit(d time.Duration, unit time.Duration) int64{
     return int64(d / unit)
}
```

Reply:
you can go a bit crazier and do
https://go.dev/play/p/syBQijmw6OW
for both functions :)

Reply:
that'll be the first instance of Generics in the repo probably. Still, we're at 1.20 now, so should be ok